test/Microsoft.Azure.WebJobs.Host.UnitTests/AttributeClonerTests.cs

```diff
@@ -251,6 +312,27 @@ public void Setting_WithNoValueInResolver_Throws()
         }
 
         [Fact]
+        public void AppSettingAttribute_DoesNotResolveBrackets()
```

Comment:
>AppSettingAttribute_DoesNotResolveBrackets [](start = 20, length = 42)

For the test here, I'd suggest:

class TestAttr { 
   [AppSetting] Prop1 ;
   [AutoResolve] Prop2;
}

Prop1 = prop2 = "test{x}and%y%"
NameResolver has x --> "somevalue", y-->"othervalue"

And then verify that Prop1 is still unchanged; and Prop2 gets the resolution. 


For the current test'
- "req{}" is not valid since the {} is empty.  We don't want somebody to think we skipped resolution because it was an error
- include the %% since Appsetting shouldn't touch those either.  Essentially, [appsetting] is a literal.